js/form-validate.js

```diff
@@ -0,0 +1,79 @@
+const Capacity = {
```

Comment:
Все же это не перечисление а мапа вспомогательная
```suggestion
const сapacityMap = {
```